src/tests/ftest/util/cont_security_test_base.py

```diff
@@ -0,0 +1,319 @@
+        if "failed" in result.stdout:
+            self.fail("    Unable to destroy pool")
+        else:
+            self.pool_uuid = None
+            self.pool_svc = None
+
+
+    def create_container_with_daos(self, pool_uuid, pool_svc, acl_type=None):
+        """Create a container with the daos tool and verify its status.
+
+        Also, obtains the container uuid from the operation's result.
+
+        Args:
+            pool_uuid (str): Pool uuid.
+            pool_svc (str): Pool service replicas.
+            acl_type (str, optional): valid or invalid.
+
+        Returns:
+            container_uuid: Container UUID created or None.
+        """
+        if not secTestBase.check_uuid_format(pool_uuid):
+            self.fail(
+                "    Invalid Pool UUID '%s' provided.", pool_uuid)
+
+        file_name = None
+        get_acl_file = None
+        expected_acl_types = [None, "valid", "invalid"]
+
+        if acl_type not in expected_acl_types:
+            self.fail(
+                "    Invalid '%s' acl type passed.", acl_type)
+
+        if acl_type:
+            get_acl_file = "acl_{}.txt".format(acl_type)
+            file_name = os.path.join(self.tmp, get_acl_file)
+        else:
+            get_acl_file = ""
+
+        if acl_type == "invalid":
+            self.daos_tool.exit_status_exception = False
+
+        kwargs = {"pool": pool_uuid,
+                  "svc": pool_svc,
+                  "acl_file": file_name}
+        container = \
+            self.daos_tool.get_output("container_create", **kwargs)
+        if len(container) == 1:
+            container_uuid = container[0]
+        else:
+            container_uuid = None
+
+        if acl_type == "invalid":
+            self.daos_tool.exit_status_exception = True
+
+        self.log.info("    daos = %s", self.daos_tool)
+
+        return container_uuid
+
+
+    @fail_on(CommandFailure)
+    def destroy_container_with_daos(self, container_uuid):
+        """Destroy a container with the daos tool and verify its status.
+
+        Args:
+            container_uuid (str): The UUID of the container to be destroyed.
+
+        Returns:
+            True or False if Container was destroyed or not.
+
+        """
+        if not secTestBase.check_uuid_format(container_uuid):
+            self.fail(
+                "    Invalid Container UUID '%s' provided.", container_uuid)
+
+        result = self.daos_tool.container_destroy(pool=self.pool_uuid,
+                                                  svc=self.pool_svc,
+                                                  cont=self.container_uuid)
+
+        self.log.info("    daos = %s", self.daos_tool)
+
+        # Verify the container destroy status
+        self.log.info("    daos.run() result =\n%s", result)
+        if "failed" in result.stdout:
+            self.log.info("    Unable to destroy container")
+            return False
+        else:
+            return True
```

Comment:
Made different updates to the code in order to use the test_utils:pool.py as well as the test_utils_container.py.
Basically the updates affected the cont_security_test_base.py where different redundant functiones were removed and others were updated.
I also had to update the "create" function in the test_utils_container.py so it can accept the acl file passed as parameter.